model/src/main/kotlin/config/HttpFileStorageConfiguration.kt

```diff
@@ -19,15 +19,26 @@
 
 package org.ossreviewtoolkit.model.config
 
+import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.databind.annotation.JsonSerialize
 import com.fasterxml.jackson.databind.util.StdConverter
 
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 data class HttpFileStorageConfiguration(
     /**
      * The URL of the HTTP server, e.g. "https://example.com/storage".
      */
     val url: String,
 
+    /**
+     * The postfix that is appended to the combination of the url and some additional path. Some storages process
+     * authentication via parameters that are within the final URL, so certain credentials can be stored in this
+     * postfix, e.g, "?user=standard&pwd=123". Thus, the final URL could be
```

Comment:
Ah, so the "postfix" is just a regular query string! Shouldn't we then also simply refer to it as such? Or are there real-world cases where this indeed needs to be a generic postfix, and *not* a query string?

Reply:
Right now I cannot think of other cases than queries. For me it would be alright to rename it. Do you prefer `query` or `queryString` ?

Reply:
I have a slight preference for just `query`.

Reply:
And also remember to reword the commit message accordingly, and BTW, also change "Url" to "URL" there, please.